testing/test_deploy.py

```diff
@@ -367,100 +422,104 @@ def deploy_minikube(args):
     if not e.content:
       raise
     content = json.loads(e.content)
-    # TODO(jlewi): We can get this error if the disk exists but not the VM. If the disk exists but not the VM
-    # and we keep going we will have a problem. However, that should be extremely unlikely now that 
-    # we set auto-delete on the disk to true.
+    # TODO(jlewi): We can get this error if the disk exists but not the VM.
+    # If the disk exists but not the vm and we keep going we will have a
+    # problem. However, that should be extremely unlikely now
+    # that we set auto-delete on the disk to true.
     if content.get("error", {}).get("code") == requests.codes.CONFLICT:
-      logging.warn("VM %s already exists in zone %s in project %s ", args.vm_name, args.zone, args.project)
+      logging.warning("VM %s already exists in zone %s in project %s ",
+                      args.vm_name, args.zone, args.project)
     else:
       raise
-  
+
   # Locate the install minikube script.
-  install_script = os.path.join(os.path.dirname(__file__), "install_minikube.sh")
-  
+  install_script = os.path.join(
+    os.path.dirname(__file__), "install_minikube.sh")
+
   if not os.path.exists(install_script):
-    logging.error("Could not find minikube install script: %s", install_script)
-    
+    logging.error("C %s", install_script)
+
   vm_util.wait_for_vm(args.project, args.zone, args.vm_name)
   vm_util.execute_script(args.project, args.zone, args.vm_name, install_script)
-  vm_util.execute(args.project, args.zone, args.vm_name, ["sudo minikube start --vm-driver=none --disk-size=40g"])
```

Comment:
we don't need to start it?

Reply:
The command happens inside install_script. It needs to happen before the commands to change the permissions.